src/terraform/structure/terraform_parser_test.go

```diff
@@ -215,7 +214,7 @@ func TestTerrraformParser_Module(t *testing.T) {
 					}
 				}
 				if !isYorTagExists {
-					t.Error(fmt.Sprintf("tag not found on merged block %v", yorTagKey))
+					t.Errorf("tag not found on merged block %v", yorTagKey)
```

Comment:
@JamesWoolfenden 😱

Reply:
ive added some new pre-commit hooks - gosimple was complaining about sprintf, fair enough?

Reply:
the white space changes are pre-commit enforcing a consistent approach to spacing and line termination.

Reply:
This is AMAZINGGGGG
Wasn't available in 1.17 I believe. this is a shock of amazement

Reply:
errorf seems so obvious i thought it was a dev mistake, didnt know it was new

Reply:
we could replace /utils but personally if it aint broke?

Reply:
we should probably update and fix all the GHA actions? Some refer to latest which we think is bad these days.

Reply:
e.g. actions/checkout@v2 where current is https://github.com/actions/checkout is v3.3

Reply:
and where i fixed the versions in the dockerfiles to FROM alpine:3.17.1